src/nginxconfig/templates/domain_sections/php.vue

```diff
@@ -219,6 +314,61 @@ THE SOFTWARE.
                 },
                 deep: true,
             },
+            // Check server selection is valid
+            '$props.data.phpServer': {
+                handler(data) {
+                    if (data.enabled) {
+                        // This might cause recursion, but seems not to
+                        if (!Object.keys(data.options).includes(data.computed))
+                            data.computed = data.default;
+
+                        // Show the custom box
+                        this.$props.data.phpServerCustom.enabled = data.computed === 'custom';
+                        return;
+                    }
+
+                    // Hide custom if disabled
+                    this.$props.data.phpServerCustom.enabled = false;
+                },
+                deep: true,
+            },
+            // Check backup server selection is valid
+            '$props.data.phpBackupServer': {
+                handler(data) {
+                    if (data.enabled) {
+                        // This might cause recursion, but seems not to
+                        if (!Object.keys(data.options).includes(data.computed))
+                            data.computed = data.default;
+
+                        // Show the custom box
+                        this.$props.data.phpBackupServerCustom.enabled = data.computed === 'custom';
+                        return;
+                    }
+
+                    // Hide custom if disabled
+                    this.$props.data.phpBackupServerCustom.enabled = false;
+                },
+                deep: true,
+            },
+            // Ensure 'Custom'/'Disabled' get translated in VueSelect on language switch
+            '$i18n.locale'() {
+                if (!this.$refs.phpServerOptions)
+                    return false;
+                const updated = this.phpServerOptions
+                    .find(x => x.value === this.$refs.phpServerOptions.$data._value.value);
+                if (updated) this.$refs.phpServerOptions.$data._value = updated;
```

Comment:
```suggestion
                if (updated) this.$refs.phpServerSelect.$data._value = updated;
```
Using the ref that will be added by the above suggestion